src/gui/editors/PianoRoll.cpp

```diff
@@ -271,6 +272,10 @@ PianoRoll::PianoRoll() :
 	{
 		s_toolOpen = new QPixmap( embed::getIconPixmap( "automation" ) );
 	}
+	if( s_toolRazor == NULL )
+	{
+		s_toolRazor = new QPixmap( embed::getIconPixmap( "razor" ) );
+	}
```

Comment:
```suggestion
	if (s_toolRazor == NULL)
	{
		s_toolRazor = new QPixmap (embed::getIconPixmap("razor"));
	}
```

Reply:
I'd add that `nullptr` should be used instead of `NULL`. That's true for the pointer declaration earlier as well.